File: 0628-maximum-product-of-three-numbers/0628-maximum-product-of-three-numbers.java
class Solution {
    public int maximumProduct(int[] nums) {
        Arrays.sort(nums);
        int n = nums.length;
        int ans = Integer.MIN_VALUE;
        ans = Math.max(ans, nums[n-1]*nums[n-2]*nums[n-3]);
        ans = Math.max(ans, nums[0]*nums[1]*nums[2]);
        ans = Math.max(ans, nums[0]*nums[1]*nums[n-1]);
        return ans;
    }
}

File: 0628-maximum-product-of-three-numbers/0628-maximum-product-of-three-numbers.cpp
class Solution {
public:
    int maximumProduct(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int n = nums.size();
        int maxSum = INT_MIN;
        maxSum = max(maxSum , nums[n-1]*nums[n-2]*nums[n-3]);
        maxSum = max(maxSum , nums[0]*nums[1]*nums[2]);
        maxSum = max(maxSum, nums[0]*nums[1]*nums[n-1]);

        return maxSum;
    }
};